proc/threads.c

```diff
@@ -1003,6 +1003,23 @@ void proc_reap(void)
 }
 
 
+void proc_changeMap(process_t *proc, vm_map_t *map, pmap_t *pmap)
+{
+	spinlock_ctx_t sc;
+
+	hal_spinlockSet(&threads_common.spinlock, &sc);
+	if (map != NULL && pmap != NULL) {
+		proc->mapp = map;
+		proc->pmapp = pmap;
+	}
+	else {
+		proc->mapp = NULL;
+		proc->pmapp = NULL;
+	}
```

Comment:
This is equivalent to simple:
```suggestion
	proc->mapp = map;
	proc->pmapp = pmap;
```

Reply:
We have no use case of changing only mapp or pmapp, both are always changed at the same time